testdata/scripts/extract.txt

```diff
@@ -552,15 +552,17 @@ msgstr ""
 msgid "Updates an account with all the fields supplied."
 msgstr ""
 -- all.md.golden --
-# Accounts API v1.0.0
+Accounts API v1.0.0
+===================
 
 Provides CRUD operations on the accounts resource.
 
-* Host `https://openbank.com`
+*   Host `https://openbank.com`
```

Comment:
markdownfmt now uses more spaces, because that is sometimes necessary by blackfriday (a kind-of buggy markdown parser, that we use in both markdownfmt and hugo)

Reply:
that looks weird, do you know why blackfriday requires double spaces here?

Reply:
Blackfriday is not commonmark compliant in weird ways. One of the weirdest issues is the spacing of lists

https://github.com/russross/blackfriday/issues/329 (3 years old issue now)

This does not parse correctly
```
* foo
  * bar
    * baz
```

this does
```
*   foo
    *   bar
        *   baz
```

this does not parse correctly
```
1. A paragraph

   item continues
```

this does (1 extra space)

```
1.  A paragraph

    item continues
```

Reply:
Hugo switched to GoldMark some time ago, but not the version we are using.

Reply:
I now switched our Hugo to the version with Goldmark, so I guess I can change it back so it uses normal two spaces :)

Reply:
I switched to Hugo everywhere in internal docs, so this is done now